File: src/Fanitty.Server.Application/Responses/Users/UserByUsernameResponse.cs
namespace Fanitty.Server.Application.Responses.Users;
public class UserByUsernameResponse
{
    public Guid Id { get; set; }
    public string Username { get; set; } = string.Empty;
    public string DisplayName { get; set; } = string.Empty;
    public string? Bio { get; set; }
}

File: client/src/app/services/responses/user-by-username-response.ts
export class UserByUsernameResponse {
    id: string;
    username: string;
    displayName: string;
    bio: string | null;

    constructor(
        id: string,
        username: string,
        displayName: string,
        bio: string
    ) {
        this.id = id;
        this.username = username;
        this.displayName = displayName;
        this.bio = bio;
    }
}